sig-security-assessments/Documentation/recipe-book.md

```diff
@@ -0,0 +1,52 @@
+# Recipe Book 👩‍🍳
+🎉Yay! You have decided to do a Self-assessment and make your corner of Kubernetes more secure! We are so happy 
+to have you here. Below is an outline of the journey to get to a successful Self-assessment.
+
+## Preparation
+Set yourself up for success – build your team and get organized
+1.	Figure out who can do this Self-assessment with you. You will need a security expert and a counterpart who owns the 
+project and is an expert in it. If you are requesting a Security Self-assessment you can reach out to the team in
+[#sig-security](https://kubernetes.slack.com/messages/sig-security) to see if there is someone interested in helping to
+champion the assessment.
+1.	Make sure that you and your counterpart each have a back-up person assigned who attends all the meetings and is 
+involved in the assessment. Two heads are better than one, and redundancy means that if someone can’t make a meeting 
+for whatever reason, that you can still make progress.
+1.	To get further participation, advertise in other SIG meetings. Aim for a minimum of 4 people (a project expert, a 
+security expert, and a backup for each), and a maximum of 8. This ensures that there is not overwhelming burden on any
+one individual but allows for a smooth decision-making process.
+1.	Create a Google doc for meeting notes for future and past meetings for tracking. Share these notes in 
+[#sig-security](https://kubernetes.slack.com/messages/sig-security), 
+[#tag-security](https://cloud-native.slack.com/archives/CDJ7MLT8S), and the appropriate Kubernetes Slack channel for 
+the project you are assessing.
+## Meetings
+- Once you have your collateral and your team together, set up a kickoff meeting with everyone. It is likely that not 
+everyone will show up. That’s ok!
+- Keep meeting notes using the 
+[Kubernetes community meeeting notes template](https://github.com/kubernetes/community/blob/master/events/community-meeting.md).
+- Create a recurring meeting series of about ~6 meetings (this is flexible and can be extended if necessary)
+- These sessions can be ad hoc and should be set for whatever timing works for the group.
+## The Assessment
+### Step One
+Draw a [data flow diagram](https://www.lucidchart.com/pages/data-flow-diagram) for the main workflows in the project. 
+This step will show you the areas you can potentially assess, and allow the security expert to quickly gain context 
```

Comment:
I'm flip flopping between the term "security champion" and "security expert" - does anyone have strong opinions one way or another?

Reply:
I think Champion is more inviting than Expert. One of the principles here that I want the subproject to espouse is that you don't have to be an expert to do something to positively augment your security posture. Having *some* security knowledge and acting on it is a good thing, in other words. Yes, we absolutely need to partner with experts to make sure that things are properly thought through, thorough, etc. But any non-expert should feel empowered to start a self-assessment and to reach out to experts for guidance, which is what we're here for.